File: src/main/java/com/tmvkrpxl0/tcombat/common/network/packets/TargetSetPacket.java
package com.tmvkrpxl0.tcombat.common.network.packets;

import javax.annotation.Nonnull;
import java.util.UUID;

public class TargetSetPacket {
    private final UUID uuid;
    private final int[] entityIds;
    public TargetSetPacket(@Nonnull UUID uuid, @Nonnull int[] entityIds) {
        this.uuid = uuid;
        this.entityIds = entityIds;
    }

    @Nonnull
    public UUID getUniqueID(){
        return this.uuid;
    }

    @Nonnull
    public int[] entityIds(){
        return this.entityIds;
    }
}

File: src/main/kotlin/com/tmvkrpxl0/tcombat/common/network/packets/TargetSetPacket.kt
package com.tmvkrpxl0.tcombat.common.network.packets

import java.util.*
import javax.annotation.Nonnull

class TargetSetPacket(
    @get:Nonnull @param:Nonnull val uniqueID: UUID,
    @param:Nonnull private val entityIds: IntArray
) {
    @Nonnull
    fun entityIds(): IntArray {
        return entityIds
    }
}